src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -126,6 +126,54 @@ public String vin() throws SmartcarException {
     return this.call("permissions", "GET", null, ApplicationPermissions.class).getData().getPermissions();
   }
 
+  /**
+   * Send request to the /permissions endpoint
```

Comment:
Wrong description right?

Reply:
It sends a request to the permissions endpoint as it fetches data from permissions, not sure if we want to keep that in or not.